src/pages/video-page/Video.jsx

```diff
@@ -1,8 +1,13 @@
 import React from "react";
 import { useContext } from "react";
 import { Aside, VideoCard } from "../../components";
-import { VideoContext, WatchLaterContext } from "../../context";
+import { PlaylistModal } from "../../components/playlist-modal/PlaylistModal";
+import {
+  VideoContext,
+  WatchLaterContext,
+} from "../../context";
 import { LikedVideoContext } from "../../context/like-video-context";
```

Comment:
You can add this import to the above context imports. Same for the `PlaylistModal` import and `useContext` as well. You are importing everything on different lines. Merge them